VK/VK/Animations/Session/NetworkService.swift

```diff
@@ -0,0 +1,125 @@
+        case getGroups
+        case searchGroups
+
+        var description: String {
+            switch self {
+            case .getFriends:
+                return Constants.getFriendsDescription
+            case .getAllPhotos:
+                return Constants.getAllPhotosDescription
+            case .getGroups:
+                return Constants.getGroupsDescription
+            case .searchGroups:
+                return Constants.searchGroupsDescription
+            }
+        }
+    }
+
+    // MARK: - Private Properties
+
+    private let friendsQueryItem = [
+        URLQueryItem(name: Constants.queryItemTokenName, value: "\(Settings.shared.token)"),
+        URLQueryItem(name: Constants.queryItemfieldsName, value: Constants.queryItemfieldsValue),
+        URLQueryItem(name: Constants.queryItemVName, value: Constants.queryItemVersionValue),
+        URLQueryItem(name: Constants.queryItemLandName, value: Constants.queryItemLandValue)
+    ]
+
+    private let allPhotosQueryItem = [
+        URLQueryItem(name: Constants.queryItemTokenName, value: "\(Settings.shared.token)"),
+        URLQueryItem(name: Constants.queryItemExtendedName, value: Constants.queryItemExtendedValue),
+        URLQueryItem(name: Constants.queryItemOwner_id, value: "-\(Settings.shared.userId)"),
+        URLQueryItem(name: Constants.queryItemVName, value: Constants.queryItemVersionValue),
+        URLQueryItem(name: Constants.queryItemLandName, value: Constants.queryItemLandValue),
+    ]
+
+    private let groupsQueryItem = [
+        URLQueryItem(name: Constants.queryItemTokenName, value: "\(Settings.shared.token)"),
+        URLQueryItem(name: Constants.queryItemExtendedName, value: Constants.queryItemExtendedValue),
+        URLQueryItem(name: Constants.queryItemVName, value: Constants.queryItemVersionValue),
+        URLQueryItem(name: Constants.queryItemLandName, value: Constants.queryItemLandValue)
+    ]
+
+    private let searchGroupsQueryItem = [
+        URLQueryItem(name: Constants.queryItemTokenName, value: "\(Settings.shared.token)"),
+        URLQueryItem(name: Constants.queryItemQName, value: Constants.queryItemQValue),
+        URLQueryItem(name: Constants.queryItemVName, value: Constants.queryItemVersionValue),
+        URLQueryItem(name: Constants.queryItemLandName, value: Constants.queryItemLandValue)
+    ]
+
+    // MARK: - Public Methods
+
+    func getFriends() {
+        getService(componentsPath: .getFriends, queryItems: friendsQueryItem)
+    }
+
+    func getAllPhotos() {
+        getService(componentsPath: .getAllPhotos, queryItems: allPhotosQueryItem)
+    }
+
+    func getGroups() {
+        getService(componentsPath: .getGroups, queryItems: groupsQueryItem)
+    }
+
+    func searchGroups() {
+        getService(componentsPath: .searchGroups, queryItems: searchGroupsQueryItem)
+    }
+
+    // MARK: - Private Methods
+
+    private func getService(componentsPath: MethodKind, queryItems: [URLQueryItem]) {
```

Comment:
```suggestion
    private func fetchData(componentsPath: MethodKind, queryItems: [URLQueryItem]) {
```
придумать название другое, что-то связанное с запросом
по текущему названию кажется что мы получаем сервис но у нас тут запрос
можно что-то со словом load или fetch